src/device_properties.cpp

```diff
@@ -105,7 +105,12 @@ struct cvk_device_properties_intel : public cvk_device_properties {
         });
     }
     std::string get_compile_options() const override final {
-        return "-hack-mul-extended -hack-convert-to-float";
+        return "-hack-mul-extended -hack-convert-to-float "
+               "-hack-image1d-buffer-bgra";
```

Comment:
Not directly related to this change but some of these limitations are presumably caused by specific drivers rather than tied to specific devices. We will probably want to capture/represent the distinction.

Reply:
I agree, that would need to be changed at some point.

Reply:
I've created https://github.com/kpet/clvk/issues/633